Mathlib/RingTheory/OreLocalization/Basic.lean

```diff
@@ -74,6 +69,8 @@ def oreEqv : Setoid (X × S) where
 end OreLocalization
 
 /-- The ore localization of a monoid and a submonoid fulfilling the ore condition. -/
```

Comment:
```suggestion
/-- The Ore localization of a monoid and a submonoid fulfilling the ore condition. -/
```

Reply:
A second Ore (in "ore" condition) that needs to be capitalized.